src/Microsoft.ML.StandardLearners/Standard/Online/OnlineGradientDescent.cs

```diff
@@ -95,22 +95,22 @@ public override LinearRegressionModelParameters CreatePredictor()
         /// <param name="featureColumn">Name of the feature column.</param>
         /// <param name="learningRate">The learning Rate.</param>
         /// <param name="decreaseLearningRate">Decrease learning rate as iterations progress.</param>
-        /// <param name="l2RegularizerWeight">L2 Regularization Weight.</param>
+        /// <param name="l2Regularization">Weight of L2 regularization term.</param>
         /// <param name="numIterations">Number of training iterations through the data.</param>
```

Comment:
```suggestion
        /// <param name="maximumNumberOfIterations">Number of training iterations through the data.</param>
```
Please also check other online learners' APIs.

Reply:
I don't see any stop rule in `OnlineLinearTrainer`.`TrainCore` but maybe it just me.

---
In reply to: [263972702](https://github.com/dotnet/machinelearning/pull/2892#discussion_r263972702) [](ancestors = 263972702)

Reply:
I am too lazy to dig into the code, but my suggestion just reflects this parameter's description.

---
In reply to: [263975389](https://github.com/dotnet/machinelearning/pull/2892#discussion_r263975389) [](ancestors = 263975389,263972702)

Reply:
But you suggest same description as I have right now....

---
In reply to: [264346723](https://github.com/dotnet/machinelearning/pull/2892#discussion_r264346723) [](ancestors = 264346723,263975389,263972702)